### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project, I laid the foundations for the Bookstore website.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import Addbook from './Addbook';
+
+const Books = () => (
+  <section className="home">
+    <h2>Books here!</h2>
+    <Addbook />
+  </section>
+);
+
+export default Books;
```
Comment (vichuge) [inline src/components/Books.js]: - [x] Good job adding the page with form on this part, now, let's add a list of books using the `Book` component. 🐶

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Addbook from './Addbook';
+
+const Books = () => (
+  <section className="home">
+    <h2>Books here!</h2>
+    <Addbook />
+  </section>
```
Comment (vichuge) [inline src/components/Books.js:10]: - [x] I see you don't have a book element, so, why not create it to show a books list on this part? 🐱

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by denisdiaconu** [comment]
Added book element and showing the list of books.

File: src/components/Books.js (hunk 1 above)
Comment (Davidosky007) [reply src/components/Books.js]: *Done 👍

File: src/components/Books.js (hunk 2 above)
Comment (Davidosky007) [reply src/components/Books.js:10]: * Done 👍

**Comment by Davidosky007** [comment]
## Hi @denisdiaconu 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
thank-you

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**